transport/src/main/java/io/netty/channel/pool/SimpleChannelPool.java

```diff
@@ -183,9 +202,9 @@ private void notifyHealthCheck(Future<Boolean> future, Channel ch, Promise<Chann
     }
 
     /**
-     * Bootstrap a new {@link Channel}. The default implementation uses {@link Bootstrap#connect()},
-     * sub-classes may override this.
-     *
+     * Bootstrap a new {@link Channel}. The default implementation uses {@link Bootstrap#connect()}, sub-classes may
+     * override this.
+     * <p/>
```

Comment:
This is not part of my changes actually. This part got auto-reformatted by IDEA based on the rules that are used in netty project. I can certainly change it to <p>